### DISCUSSION TITLE
Randomize window position on Almost-Maximize

### DISCUSSION THREAD

**Discussion by Myridium** [rxhanson/Rectangle#1069] [Ideas]
It's a lot easier to get a grip on how many windows are open, and where they are, when they are not overlapping in a pixel-perfect way. I think it would be a great improvement if there were an option to perform "Almost Maximize", except the position of the window is randomized. (The window should still fit fully on the screen.)

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't think I'll add something like this in, but feel free to fork it. I can tell you where to find things if you have trouble getting started.
